core/modules/installer/installer.browser.pages.inc

```diff
@@ -0,0 +1,884 @@
+    menu_rebuild();
+  }
+}
+
+/**
+ * Form builder for the select versions form.
+ *
+ * @param $projects
+ *   An array of projects selected for install, keyed by project name.
+ */
+function installer_browser_installation_enable_form($form, &$form_state, $projects) {
+  $modules = system_rebuild_module_data();
+
+  $options = array();
+  $missing = array();
+
+  foreach ($projects as $project) {
+    if ($project['type'] == 'module') {
+      $dependency_check = TRUE;
+      $dependencies = array();
+      if (isset($modules[$project['name']])) {
+        foreach ($modules[$project['name']]->info['dependencies'] as $dependency) {
+          if (isset($modules[$dependency])) {
+            $dependencies[] = $modules[$dependency]->info['name'] . ' (' . t('Installed') . ')';
+          }
+          else {
+            $dependency_check = FALSE;
+            $dependencies[] = $dependency . ' (' . t('Missing') . ')';
+          }
+        }
+        if ($dependency_check) {
+          $options[$project['name']] = array(
+            array('data' => $modules[$project['name']]->info['name']),
+            array('data' => $modules[$project['name']]->info['version']),
+            array('data' => implode(', ', $dependencies)),
+          );
+        }
+        else {
+          $missing[$project['name']] = array(
+            array('data' => $modules[$project['name']]->info['name']),
+            array('data' => $modules[$project['name']]->info['version']),
+            array('data' => implode(', ', $dependencies)),
+          );
+        }
+      }
+      else {
+        backdrop_set_message(t('There was an error getting information for @module',
+          array('@module' => $project['name'])), 'error');
+      }
+    }
+  }
+
+  $headers = array(
+    array('data' => t('Title')),
+    array('data' => t('Version')),
+    array('data' => t('Dependencies')),
+  );
+
+  if(!empty($options) || !empty($missing)) {
+    $form['module_instructions'] = array(
+    '#type' => 'item',
+    '#markup' => t('You may enable modules using the form below or on the main !link page.', array('!link' => l(t('Modules'), 'admin/modules'))),
+    );
+    if (!empty($options)) {
+      $form['modules'] = array(
+        '#type' => 'tableselect',
+        '#title' => t('Enable modules'),
+        '#description' => t('Select which modules you would like to enable.'),
+        '#header' => $headers,
+        '#options' => $options,
+        '#empty' => t('No new modules installed.'),
+        '#multiple' => TRUE,
+        '#js_select' => TRUE,
+        '#weight' => 1,
+      );
+
+      $form['actions'] = array('#type' => 'actions');
+      $form['actions']['submit'] = array(
+        '#type' => 'submit',
+        '#submit' => array('installer_browser_installation_enable_form_submit'),
+        '#value' => t('Enable modules'),
+      );
+      $form['actions']['cancel'] = array(
+        '#type' => 'link',
+        '#name' => 'cancel',
+        '#title' => t('Cancel'),
```

Comment:
This link should read 'Skip'.  We are not cancelling the installation, just skipping the enable step.